src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  if (days < 3) {
+    return days * 40;
```

Comment:
Consider using a descriptive constant for the daily rental rate instead of the hardcoded value `40`. This will make your code more maintainable and easier to understand.